Sources/MapboxCoreNavigation/PassiveLocationManager.swift

```diff
@@ -199,21 +203,34 @@ open class PassiveLocationManager: NSObject {
         }
     }
     
+    /**
+     Starts recording history for debugging purposes.
+     
+     - postcondition: Use the `stopRecordingHistory(writingFileWith:)` method to stop recording history and write the recorded history to a file.
+     */
+    public static func startRecordingHistory() {
+        Navigator.shared.startRecordingHistory()
+    }
+    
     /**
      A closure to be called when history writing ends.
      
      - parameter historyFileURL: A path to file, where history was written to.
```

Comment:
It would be better to describe that the `historyFileURL` could be `nil`, e.g. if history recording hadn't been started before the `stopRecordingHistory` was called.